tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -770,64 +776,80 @@ private Properties getProperties( File file )
 
     public Map<String,Object> getBuildProperties() throws CoreException
     {
+        return getBuildProperties(false,true);
+    }
+
+    public Map<String,Object> getBuildProperties(final boolean reload, final boolean verified) throws CoreException
```

Comment:
It seems that we don't need both booleans.  Because I never see you called it with all possible combinations.  Seems it is either not used or both are true.